### ISSUE TITLE
In Thunderbird, when navigating to a line beginning with a labelled link, the url is displayed on the braille display instead of the label

### ISSUE THREAD

**Issue by Hatoon-fel** [nvaccess/nvda#16469]
### Steps to reproduce:
Open NVDA browse mode settings With NVDA+CTRL+B.
Check the Check boxe: "Automatically set system focus to focusable elements".
open any message in thhunderbird's mesages list (e.g inbox) that contains html content with labeled link, such as the emails of notifications from GitHub.
arrow down to any line containing labeled link at beggining, for example: issue number.
arrow up then arrow down to the same link again or simply press home button to go to the beggining of the line.
### Actual behavior:
On Braile display when focus on the link for the first time, The URl is displayed instead the label of that link, not as spoken text, as nvda reads the label then the url. also, not as in Windows Mail and Calendar app.

### Expected behavior:
NVDA displays the label of the link on Braile as speech, and additionally, i think, it's not proper behavior reading the urlK as same as in Windows mail app.

#### NVDA installed/portable/running from source:
Installed and portable.
#### NVDA version:
NVDA2024.1 and Alpha31645
#### Windows version:
Windows version: 10.0.19045 Built 19045	
#### Name and version of other software in use when 
Thunder Bird 115.10.1
reproducing the issue:

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

**Comment by CyrilleB79** [comment]
Does the issue still occur if you uncheck "Automatically set system focus to focusable elements"? If not, why do you need to check this checkbox rather than keeping the default unchecked value?

**Comment by burmancomp** [comment]
Is original or simple html selected from "message body as" submenu in view submenu?

**Comment by Hatoon-fel** [comment]
@CyrilleB79 No, when i uncheck this checkbox, every thing work as expected. But, When i faced this issue, The option was activated, so, I  thoght that is the default, Maybe I activated before for trying, and i thoght if it is enabled it will provide better experience.

**Comment by Hatoon-fel** [comment]
@burmancomp Yes, I tried with both of them

**Comment by Adriani90** [comment]
No, enabling that setting degradates the user experience and actually NV Access plans to remove it completely.
@seanbudd , @michaelDCurran is there a timeline for this?

**Comment by seanbudd** [comment]
@Adriani90 - we don't have a clear date for this yet but it is on the roadmap
